### ISSUE TITLE
The ship in Eliminator leaves stray pixels behind.

### ISSUE THREAD

**Issue by Keatah** [AppleWin/AppleWin#606]
The blue/player ship in Eliminator leaves stray pixels behind. This seems to happen with the following settings under the INPUT tab:

Joystick 1: Keyboard(cursors)
[checked] Allow cursor keys to be read from keyboard

Unchecking it seems to correct the issue. At first I thought it was the "COLOR SIMPLIFIED" option. But no. I think the problem goes back to 1.25.0.4 or earlier.

The disk in question is:
ftp.apple.asimov.net\pub\apple_II\images\games\action\eliminator.dsk

Start the game using Keyboard.

**Comment by tomcw** [comment]
Just  a thought, but I've not checked this...

The down/up cursors are equivalent to CTRL+J/CTRL+K respectively.
Perhaps the game uses CTRL+J/K to (eg) enable joystick/keyboard (like Lode Runner), so it could be confusing to the game to be simultaneously toggling joystick/keyboard and moving the joystick down/up.

Can you try unchecking the "Allow cursor keys to be read from keyboard"?

**Comment by tomcw** [comment]
OK, I checked this and it looks like a bug in the game.

Regardless of starting the game with (K)eyboard or (P)addles, it always checks these keys at $8D2:

$C000 key value | Action
-|-
9B? | ESC = pause
D3? | 'S' = switch direction
C4? | 'D' = fire
C1? | 'A' = accelerate
95? | CTRL-U = right arrow
88? | CTRL-H (backspace) = left arrow

If you start the game using (K)eyboard, then cursor left/right = ship down/up.
If you start the game using (P)addles, then cursor left/right still moves the ship down/up **and** if used for joystick emulation it'll also move the ship left/right!

I think the game just can't copy with simultaneously doing both for the ship.

I tried the game in a few other emulators (eg. Apple in PC) and I got the same buggy behaviour.

I'm closing this as invalid.
